src/assets/upload.vue

```diff
@@ -0,0 +1,6 @@
+<template>
+    <svg width="29" height="28" viewBox="0 0 49 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can be reduced to 
`<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 49 48"><path fill="#fff" d="M37.34 16.44a14 14 0 0 0-26.72 3.78A8 8 0 0 0 12.5 36a2 2 0 0 0 0-4 4 4 0 0 1 0-8 2 2 0 0 0 2-2 10 10 0 0 1 19.46-3.22 2 2 0 0 0 1.56 1.34A6 6 0 0 1 36 31.8a2.004 2.004 0 1 0 1 3.88 10 10 0 0 0 .34-19.24Zm-11.42 4.14a2 2 0 0 0-.66-.42 2 2 0 0 0-1.52 0 2 2 0 0 0-.66.42l-6 6a2.008 2.008 0 1 0 2.84 2.84l2.58-2.6V38a2 2 0 0 0 4 0V26.82l2.58 2.6a2 2 0 0 0 2.84 0 2.003 2.003 0 0 0 .438-2.188 2.002 2.002 0 0 0-.438-.651l-6-6Z"/></svg>`